internal/sidekick/parser/protobuf.go

```diff
@@ -708,6 +714,19 @@ func addEnumDocumentation(state *api.APIState, p []int32, doc string, eFQN strin
 	}
 }
 
+func parseResourcePatterns(patterns []string) [][]api.PathSegment {
+	var parsedPatterns [][]api.PathSegment
+	for _, p := range patterns {
+		tmpl, err := httprule.ParseResourcePattern(p)
+		if err != nil {
+			slog.Warn("failed to parse resource pattern", "pattern", p, "error", err)
```

Comment:
Why only a warning?  This seems like a really bad problem in the input.  Wouldn't we prefer to stop the generation and fix the input?